viewmaster/viewmaster.py

```
@@ -1,6 +1,27 @@
+render HTML.
+
+Environment
+  - `PDS3_HOLDINGS_DIR`: Absolute path to the PDS3 holdings root.
+
+Configuration
+  Values are imported from `viewmaster_config.py` to configure logging,
+  filesystem locations, caching, and URL prefixes.
+"""
+
+from tkinter.constants import ON
 from flask import Flask, flash, redirect, render_template, redirect, request, send_file
```

Ruff (0.14.5)

Redefinition of unused `redirect` from line 21

Remove definition: `redirect`

(F811)